src/components/map/components/legend/config/index.js

```diff
@@ -0,0 +1,64 @@
+import { CATEGORIES, TYPES } from '../../../../../../.storybook/utils';
+import { LEGEND_PLACEMENT } from '../enums/index.js';
+
+export const argTypes = {
+  placement: {
```

Comment:
Ik vraag me af of het wel nodig/gewenst is dat je mensen toelaat om te bepalen waar de legende moet komen. Dan wijkt de stijl (als je het zo mag noemen) van individuele applicaties van elkaar af op dat vlak.

Reply:
Er is hiervoor gekozen omdat iedere applicatie een verschillend aantal items in de legende kan hebben. Afhankelijk van het aantal items in de legende zou een andere positie van de legende op de kaart beter uitkomen. Ik had ook nog gedacht de optie te maken dat de legende items verticaal of horizontaal zouden getoond kunnen worden in de legende maar zover ben ik dus niet geraakt en worden de items in de legende enkel horizontaal getoond. Is mss nog een mogelijke uitbreiding naar de toekomst toe.

Reply:
Ok, zolang het een bewuste en doordachte keuze is dat die plaatsing "vrij" is, heb ik daar zeker niets op tegen. Het is ook eerder iets voor @kspeltix @meirege om over te beslissen, ik wou het ballonnetje gewoon even voor de zekerheid oplaten.